docs/io.cozy.photos.md

```diff
@@ -0,0 +1,41 @@
+[Table of contents](README.md#table-of-contents)
+
+# Cozy Photos doctype
+
+## `io.cozy.photos.albums`
+
+This doctype is used both for manual albums and for clusters.
+
+- `name`: {string} the name of the album. It is the date of the oldest photo for a cluster.
+- `auto`: {bool} `true` if the album was automatically computed, and is therefore a cluster.
+- `period`: {object} if cluster, the temporal period of the album.
+  - `start`: {date} the date of the oldest photo.
+  - `end`: {date} the date of the newest photo.
+
+The photos in an album are files (with the class `image`) that are [referenced](https://cozy.github.io/cozy-stack/references-docs-in-vfs.html) by the album document.
+
+## `io.cozy.photos.settings`
+
+This doctype is used to store settings for photos, useful for the clustering.
+
+- `type`: {string} the type of settings, e.g. `clustering`.
+
+### Clustering settings
+
+- `lastDate`: {date} the date of the last clustered photo (based on `created_at`).
+- `jobStatus`: {string} the execution status, can be `running` if the service is running, `postponed` if an execution is planned later, or empty.
+- `lastExecution`: {timestamp} the last service execution timestamp. Useful when `jobStatus` is `postponed`, to know if the service should be run or not.
```

Comment:
Why not a date ?
Using timestamps is a bad habit.

Reply:
It is the result of `Date.now()` to compute easily the difference with another `Date.now()` ; why is it a bad habit?

Reply:
You can't have a timezone, you can't go after 2038 on most systems, and you have multiple ways to deal with dates storage (since other properties will have to deal with one of these two other constraints even if this one may not)

Reply:
Ok fair enough, thanks!